macos/library/FLEFileChooserPlugin.m

```diff
@@ -96,6 +121,9 @@ - (void)configureOpenPanelForClient:(nonnull NSNumber *)clientID
     ((NSOpenPanel *)_panels[clientID]).canChooseDirectories =
         [arguments[kCanChooseDirectoriesKey] boolValue];
   }
+  if ([argKeys containsObject:kCanChooseFilesKey]) {
+    ((NSOpenPanel *)_panels[clientID]).canChooseFiles = [arguments[kCanChooseFilesKey] boolValue];
```

Comment:
(NSOpenPanel *)_panels[clientID] is being repeated enough times now that it should be extracted to a new local variable at the start of the method.

Reply:
Done.